geom.go

```diff
@@ -193,6 +193,19 @@ func (l Layout) ZIndex() int {
 	}
 }
 
+// TransformInPlace replaces all coordinates in g using f.
+func TransformInPlace(g T, f func(Coord, Layout, int)) T {
+	var (
+		layout     = g.Layout()
+		stride     = g.Stride()
+		flatCoords = g.FlatCoords()
+	)
+	for i, n := 0, len(flatCoords); i < n; i += stride {
+		f(flatCoords[i:i+stride], layout, stride)
```

Comment:
What's the purpose of passing the stride?

Reply:
You're right. As the length of the `Coord` passed to f is equal to `stride`, there is no need to pass the stride.